ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```
@@ -137,15 +100,13 @@ else
 
-if [[ -n "$ORION_CONFIG" ]]; then
-    if [[ "$ORION_CONFIG" =~ ^https?:// ]]; then
-        fileBasename="$(basename ${ORION_CONFIG})"
-        if curl -fsSL "$ORION_CONFIG" -o "$ARTIFACT_DIR/$fileBasename"; then
-            ORION_CONFIG="$ARTIFACT_DIR/$fileBasename"
-        else
-            echo "Error: Failed to download $ORION_CONFIG" >&2
-            exit 1
-        fi
+if [[ -n "${ORION_CONFIG}" ]] && [[ "$ORION_CONFIG" =~ ^https?:// ]]; then
+    fileBasename="$(basename ${ORION_CONFIG})"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)